## Methods

Dogs were recruited by word of mouth in several locations between July and November 2022.
Breed organization registration of each dog was confirmed by manually viewing the dog's breed organization paperwork or locating the dog in the American Kennel Club (AKC) database.
Recruitment was run to ensure full coverage of dog breeds across breed clades [@doi:10.1016/j.celrep.2017.03.079].
Each dog was randomized to either the control or treatment group, corresponding to the dog's own versus a randomized photo.
Dogs in the treatment group were paired based on their enrollment order, and the photographs were swapped across pairs.
Assignments were evaluated to ensure that dogs within a breed clade (per [@doi:10.1016/j.celrep.2017.03.079]) were not paired.
All enrolled dogs came from different households, avoiding the possibility of cross-contamination of dogs within the study.

We evaluated six tests on the market at the end of 2021, when the study began.
DTC genetic tests were purchased directly from each company's website.
Kits were registered according to the instructions provided, using the dog's own photograph (control) or the photograph of a paired dog (treatment).
The kits were then delivered to the owner, who was instructed to collect buccal swabs for each of the 6 kits in Table {@tbl:tests} according to manufacturer instructions and mail the samples using the company-provided packaging.
Genetic testing was conducted by each DTC genetic testing company according to their own protocols.
The breed determination results were returned to the authors, who retrieved the results for each dog and recorded all results for the determined breeds and the corresponding percentage composition in a spreadsheet.

**Comparison of six DTC dog genetic testing services.**
At the time of study design (end of 2021), six DTC dog genetic tests were available on the public market.
All 6 were included in this study.
Each test uses a distinct combination of genotyping technologies, breed classification algorithms, and reference panels (see {#tbl:tests}).
While this study focuses on the methodologies, a detailed breakdown of consumer-oriented similarities and differences among several of these tests is available elsewhere [@url:https://www.caninejournal.com/dog-dna-tests-reviews].
{#tbl:tests}

| Test/Company Name                               | Price of Base Breed Test   | Markers Used                                                                                                                    | Reference Panel                                                                                                           | Ancestry Assignment Algorithm             |
|:------------------------------------------------|:---------------------------|:--------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------|:------------------------------------------|
| Embark Breed ID DNA Test [@url:https://shop.embarkvet.com/products/embark-dog-dna-test-kit]       | $129-$199                  | 200,000+ SNPs, custom chip [@doi:10.1371/journal.pgen.1007648; @doi:10.1371/journal.pgen.1008003] including all markers 173k markers on the Illumina CanineHD platform [@doi:10.1534/g3.118.200836]                                                                       | 350+ breeds [@url:https://embarkvet.com/resources/dog-breeds]                                                                                                          | Not specified                             |
| Wisdom Panel Breed Discovery [@url:https://www.wisdompanel.com/en-us/dog-dna-tests/breed-discovery]                   | $80-$160                   | SNPs, number not specified, custom-designed Illumina Infinium XT microarray [@doi:10.1371/journal.pgen.1010651]                                                                     | 350+ breeds, 21,000+ samples [@url:https://www.wisdompanel.com/en-us/our-science; @doi:10.1371/journal.pgen.1010651] | BCSYS [@doi:10.1371/journal.pgen.1010651]                                     |
| Orivet GenoPet [@url:https://www.orivet.com/us/products/genopet-dog-breed-identification-dna-test]                                  | $95-160                    | SNPs, number not specified [@url:https://www.orivet.com/store/canine-mixed-breed-screen/geno-pet-dog-breed-identification-test] | 320+ breeds [@url:https://www.orivet.com/store/breed-list]                                                                | Not specified                             |
| Darwin's Ark Explorer Level Donation [@url:https://darwinsark.org]           | $149 donation              | Call 9M SNPs from whole-genome sequencing, use 688K for breed classification [@url:https://darwinsark.org/faqs] | 101 breeds [@url:https://darwinsark.org/faq]                                                                                                               | SupportMix [@doi:10.1186/1471-2156-13-49] |
| DNA My Dog Essential Breed ID Test PLUS Wolf - Canid/Hybrid Test [@url:https://dnamydog.com/tests]             | $80-$130                   | Copy-number variation, number of markers not specified [@url:https://dnamydog.com/help/help-centre]                             | 350+ breeds [@url:https://dnamydog.com/science/breeds-we-test]                                                                                                              | Not specified                             |
| Accu-Metrics/Via-Pet Canine Breed Determination [@url:https://www.via-pet.com/canine-testing/p/dog-breed-identification] | $59                        | Not specified                                                                                                                   | 340 breeds [@url:https://www.via-pet.com/canine-testing/p/dog-breed-identification]                                       | Not specified                             |


As a positive control for the potential photographic test bias, we evaluated each sample/photograph pair using a pre-trained convolutional neural network, to predict breed.
Dog breed classification is a classic problem in computer vision, and models trained on images from a large compendium (ImageNet) have been evaluated for their performance in classifying photographs of purebred dogs, with the model NASNet showing particularly robust performance [@doi:10.1007/s11633-020-1261-0; @doi:10.1109/sisy.2018.8524715; @doi:10.1109/iciteed.2019.8929955].
Therefore, NASNet complements DTC genetic testing results by predicting breeds based exclusively on photographs.
We loaded the NASNet model **nasnetalarge** from the Python package PyTorch Image Models (timm version 0.6.12) [@url:https://github.com/rwightman/pytorch-image-models] and used to classify each of the dog photos.<!-- acknowledge guide from https://rwightman.github.io/pytorch-image-models/models/inception-resnet-v2-->
Because the classifier estimates many categories with a very long tail, only hits with a score greater than 1% were included in the analysis.

To evaluate whether swapping the photographs affected breed prediction for each DTC test, the proportion of results matching the registered breed across the two conditions was compared by calculating the odds ratio.
In cases where a test had either zero matching or zero non-matching predictions within a condition (i.e., one or more zero cells), the Haldane-Anscombe correction [@doi:10.1081/sac-200040691] was applied per [@doi:10.1002/jrsm.1460; @doi:10.15678/znuek.2015.0941.0504].
We evaluated independence between the treatment and control conditions using a one-tailed hypergeometric test.
Rejecting the null would indicate a potential relationship between the treatment (photograph matching status) and outcome.
Both statistical tests were conducted in R using the package vcd [@url:https://CRAN.R-project.org/package=vcd].

Based on qualitative patterns observed in the results, the decision was made *post hoc* to compare the phylogenetic clades [@doi:10.1016/j.celrep.2017.03.079] of predicted breeds to those of the registered breeds of the DNA donor and photographed dogs.
Each breed was mapped onto a clade, as identified by [@doi:10.1016/j.celrep.2017.03.079] or based on its close genetic relationship to breeds assigned to clades in [@doi:10.1016/j.celrep.2017.03.079].
In the treatment condition, each predicted breed's clade was compared to the clades of the DNA donor versus the dog whose photograph was provided to the testing company.
In the control condition, each dog was paired with another dog (based on the order in which they were recruited).
The breed predictions were then evaluated to determine whether their clade matched the clade of the photographed or paired dog (in the treatment and control conditions, respectively).
The goal of this analysis was to evaluate whether results that matched neither the registered breed of the DNA donor nor the photograph provided were closely related to, but not identical to, the photographed dog.
The odds ratio and hypergeometric tests were applied using the vcd package in R.
Additionally, the results at the level of breed clade were plotted in R using the ternary package [@url:https://CRAN.R-project.org/package=Ternary].
Qualitative results (e.g., range-format percentages) were converted to numerical values for the purposes of visualizing the results (Supplementary Table XX).

