## Methods

Dogs were recruited by word of mouth between July and November 2022.
Registration eligibility of each dog was confirmed by manually viewing the dog's breed organization paperwork or locating the dog in the American Kennel Club (AKC) database.
Recruitment was structured to ensure full coverage across dog breed clades [@doi:10.1016/j.celrep.2017.03.079].
As our goal was to evaluate the influence of photographs, we divided dogs into two groups.
We considered these groups a control condition, in which the photograph matched the dog, and a treatment condition, in which the photograph was swapped from a different dog in the group.
Thus, the term treatment refers to the photo and no treatment was applied to the dogs themselves.
Randomization was performed using the using the RANDBETWEEN() function in Excel.
All enrolled dogs came from different households.

We selected DTC tests that indicated an ability to predict breed that the authors identified as available in 2021, when the study was initiated, and that we were able to purchase through our institutional purchasing systems or processes.
DTC genetic tests (Table @tbl:tests) were purchased directly from each company's website.
Kits were registered according to the instructions provided, using the dog's own photograph (control) or the photograph of a dog paired based on enrolment order (treatment).
When the information was requested, dogs were listed as mixed breed.
For some companies, this may support a search for additional minor-breed signature differences.
Each dog owner, who was instructed to collect buccal swabs for each kit according to manufacturer instructions and to mail the samples using the company-provided packaging.
Genetic testing was conducted by each company according to their own protocols.
The breed determination results were returned to the investigators and harmonized across studies using the Vertebrate Breed Ontology version 20230601 [@doi:10.5281/zenodo.7996675; @icbo-vbo].
The VBO was created to provide a standardized resource of breed names of many animals, including dogs.
If a breed assignment corresponded to a variety, rather than a breed, per the Federation Cynologique Internationale (FCI) [@ALyYdaeN], the corresponding breed was used instead.
Qualitative assignments, i.e, ranges of percentages, were converted to numerical values (Supplementary Table 2).

| Company | Markers Used | Reference Panel | Ancestry Assignment Algorithm |
|:--------|:-------------|:----------------|:------------------------------|
| Embark [@url:https://shop.embarkvet.com] | 200,000+ SNPs, microarray [@1CDZjpw5n; @doi:10.1371/journal.pgen.1008003] | 350+ breeds [@doi:10.1534/g3.118.200836] | unknown|
| Wisdom Panel [@url:https://www.wisdompanel.com] | SNPs, microarray [@doi:10.1371/journal.pgen.1010651] | 350+ breeds, 21K+ samples [@doi:10.1371/journal.pgen.1010651; @url:https://www.wisdompanel.com] | BCSYS [@doi:10.1371/journal.pgen.1010651] |
| Orivet [@url:https://www.orivet.com] | SNPs [@url:https://www.orivet.com] | 350+ breeds, 15K samples [@url:https://www.orivet.com] | unknown |
| Darwin's Ark [@url:https://darwinsark.org] | 688K SNPs, whole-genome sequencing [@url:https://darwinsark.org] | 101 breeds [@url:https://darwinsark.org] | SupportMix [@doi:10.1186/1471-2156-13-49; @doi:10.1126/science.abk0639] |
| DNA My Dog [@url:https://dnamydog.com] | Copy-number variants [@url:https://dnamydog.com] | 350+ breeds [@url:https://dnamydog.com] | unknown |
| Accu-Metrics [@url:https://www.via-pet.com] | unknown | 340 breeds [@url:https://www.via-pet.com] | unknown |
Table:
Comparison of DTC canine ancestry identification services.
At the time of study design (end of 2021), six tests were commercially available.
{#tbl:tests}

As a positive control for use of the photograph, we also analyzed each sample/photograph pair using a pre-trained convolutional neural network using a model NASNet, that has shown particularly robust performance for dog breed identification [@doi:10.1007/s11633-020-1261-0; @doi:10.1109/SISY.2018.8524715; @doi:10.1109/ICITEED.2019.8929955].
We loaded the NASNet model **nasnetalarge** from the Python package PyTorch Image Models (timm version 0.6.12)[@doi:10.1007/s11633-020-1261-0; @doi:10.1109/ICITEED.2019.8929955] and used it to it to classify each of the dog photos, adapting code from a tutorial [@url:https://github.com/huggingface/pytorch-image-models].
The classifier estimates many categories with a very long tail, so only hits with a score >1% were included in the analysis.

To evaluate the effect of the photograph on each test's predictions, the proportion of results matching the registered breed was evaluated using an odds ratio in the R package vcd [@doi:10.18637/jss.v017.i03].
The Haldane-Anscombe correction [@doi:10.1081/SAC-200040691] was applied when appropriate [@doi:10.1002/jrsm.1460].
Independence was assessed with a one-tailed hypergeometric test.
Rejection of the null, corresponding to a p-value of < 0.05,  would indicate a potential relationship between condition and outcome.

In a *post hoc* analysis, we also compared the results to the DNA donor and photograph at the level of breed clade [@doi:10.1016/j.celrep.2017.03.079].
For breeds not assigned to a clade [@doi:10.1016/j.celrep.2017.03.079], a determination was made based on the closest genetic relatives assigned.
Each predicted breed was classified based on whether it matched the DNA donor, the paired dog's breed (in the treatment condition, this was also the photograph), or neither.
Odds ratios and the p-values associated with the hypergeometric tests were calculated in R using vcd, and the results were visualized using ternary [@doi:10.5281/zenodo.7924133].

