## Methods

Dogs were recruited by word of mouth in several locations between July and November 2022.
Breed organization registration eligibility of each dog was confirmed by manually viewing the dog's breed organization paperwork or locating the dog in the American Kennel Club (AKC) database.
Recruitment was structured to ensure full coverage across dog breed clades [@doi:10.1016/j.celrep.2017.03.079].
Each dog was randomized to either the control or treatment group, corresponding to the dog's own photo versus a randomized photo.
Dogs in the treatment group were paired based on their enrollment order, and the photographs were swapped across pairs.
Assignments were evaluated to ensure that dogs within a breed clade (per Parker et al. [@doi:10.1016/j.celrep.2017.03.079]) were not paired.
All enrolled dogs came from different households, avoiding the possibility of cross-contamination of dogs within the study.

The six DTC genetic tests (Table @tbl:tests) were purchased directly from each company's website.
Kits were registered according to the instructions provided, using the dog's own photograph (control) or the photograph of a paired dog (treatment).
When the information was requested, dogs were listed as mixed breed.
A set of kits was then delivered to each dog owner, who was instructed to collect buccal swabs for each according to manufacturer instructions and to mail the samples using the company-provided packaging.
Genetic testing was conducted by each DTC genetic testing company according to their own protocols.
The breed determination results were returned to the authors, who retrieved the results for each dog and recorded all results for the determined breeds and the corresponding percentage composition in a spreadsheet.
Breed results were harmonized across studies using the Vertebrate Breed Ontology version 20230601 [@doi:10.5281/zenodo.7996675; @icbo-vbo].
If a breed assignment corresponded to a variety, rather than a breed, per the Federation Cynologique Internationale (FCI) [@ALyYdaeN], the corresponding breed (i.e., the parent term) was used instead.
Qualitative assignments, such as ranges of percentages, were converted to numerical values for the purposes of data visualization (Supplementary Table 2).

| Name of Breed Test                                                                                                       | Price of Available Test   | Markers Used                                                                                                                                                   | Reference Panel                                                                                                                                       | Ancestry Assignment Algorithm           |
|:------------------------------------------------------------------------------------------------------------------------|:---------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------|
| Embark Breed ID DNA Test [@url:https://shop.embarkvet.com]                              | $129-$199                  | 200,000+ SNPs, microarray [@1CDZjpw5n; @doi:10.1371/journal.pgen.1008003] | 350+ breeds [@doi:10.1534/g3.118.200836]                                                                                                              | Not specified                           |
| Wisdom Panel Breed Discovery [@url:https://www.wisdompanel.com]                      | $85-$160                   | SNPs, microarray [@doi:10.1371/journal.pgen.1010651]                                                       | 350+ breeds, 21,000+ samples [@doi:10.1371/journal.pgen.1010651; @url:https://www.wisdompanel.com] | BCSYS [@doi:10.1371/journal.pgen.1010651]      |
| Orivet GenoPet [@url:https://www.orivet.com]                                                                          | $95-160                    | SNPs [@url:https://www.orivet.com]                                 | 15,000 samples from 350+ breeds [@url:https://www.orivet.com]                                | Not specified                           |
| Darwin's Ark Explorer Level Donation [@url:https://darwinsark.org]                                                       | $149 donation              | 688K SNPs, whole-genome sequencing [@url:https://darwinsark.org]                                                 | 101 breeds [@url:https://darwinsark.org]                                                                                                          | SupportMix [@doi:10.1186/1471-2156-13-49; @doi:10.1126/science.abk0639] |
| DNA My Dog Essential Breed ID Test PLUS Wolf - Canid/Hybrid Test [@url:https://dnamydog.com]                       | $80-$130                   | Copy-number variation, number of markers not specified [@url:https://dnamydog.com]                                                             | 350+ breeds [@url:https://dnamydog.com]                                                                                         | Not specified                           |
| Accu-Metrics/Via-Pet Canine Breed Determination [@url:https://www.via-pet.com] | $59                        | Not specified                                                                                                                                                  | 340 breeds [@url:https://www.via-pet.com]                                                                    | Not specified                           |

Table: Comparison of six DTC dog genetic testing services.
At the time of study design (end of 2021), six DTC dog genetic tests were commercially available.
All six were included in this study.
Each test uses a distinct combination of genotyping technologies, breed classification algorithms, and reference panels.
While this study focuses on the methodologies, a detailed breakdown of consumer-oriented similarities and differences among several of these tests is available elsewhere [@url:https://www.caninejournal.com/dog-dna-tests-reviews].
{#tbl:tests}

As a positive control for the potential photographic test bias, we predicted the breed of each sample/photograph pair using a pre-trained convolutional neural network.
Dog breed classification is a classic problem in computer vision, and models trained on images from a large compendium (ImageNet) have been evaluated for their performance in classifying photographs of purebred dogs.
The model NASNet shows particularly robust performance [@doi:10.1007/s11633-020-1261-0; @doi:10.1109/SISY.2018.8524715; @doi:10.1109/ICITEED.2019.8929955].
Therefore, we used NASNet predictions to complement the DTC genetic testing results by predicting breeds based exclusively on photographs.
We loaded the NASNet model **nasnetalarge** from the Python package PyTorch Image Models (timm version 0.6.12)[@url:https://doi.org/grwfjc; @url:https://doi.org/grwfjf] and used it to it to classify each of the dog photos, adapting code from a tutorial [@url:https://github.com/huggingface/pytorch-image-models].<!--Halie fix this-->
The classifier estimates many categories with a very long tail, and so only hits with a score >1% were included in the analysis.

To evaluate whether swapping the photographs affected breed prediction for each DTC test, the proportion of results matching the registered breed across the two conditions was compared by calculating the odds ratio.
In cases where there were either zero matching or zero non-matching predictions within a condition (i.e., one or more zero cells), the Haldane-Anscombe correction [@doi:10.1081/SAC-200040691] was applied per [@doi:10.1002/jrsm.1460; @doi:10.15678/ZNUEK.2015.0941.0504].
We evaluated independence between the treatment and control conditions using a one-tailed hypergeometric test.
Rejection of the null would indicate a potential relationship between the treatment (photograph matching status) and outcome.
Both statistical tests were conducted in R using the package vcd [@doi:10.18637/jss.v017.i03; @doi:10.1198/106186007X237856].

Based on qualitative patterns observed in the results, the decision was made *post hoc* to compare the phylogenetic clade [@doi:10.1016/j.celrep.2017.03.079] of each predicted breed against the registered breeds of the DNA donor and photographed dog.
Each breed was mapped onto a clade, as identified by Parker et al. [@doi:10.1016/j.celrep.2017.03.079] or, for breeds that were not included in that analysis, based on its closest genetic relationship to breeds assigned to clades.
In the treatment condition, each predicted breed's clade was compared to the clades of the DNA donor versus the dog whose photograph was provided to the testing company.
In the control condition, each dog was paired with another dog (based on the order in which they were recruited) to provide a random baseline.
The breed predictions were then evaluated to determine whether their clade matched the clade of the photographed or paired dog in the treatment and control conditions, respectively.
The goal of this analysis was to evaluate whether predicted breeds that matched neither the registered breed of the DNA donor nor the photograph were closely related to but not identical to either dog breed.
The odds ratio and hypergeometric tests were applied using the vcd package in R. Additionally, the results at the level of breed clade were plotted in R using the ternary package [@doi:10.5281/zenodo.7924133].
Qualitative results (e.g., range-format percentages) were converted to numerical values for the purposes of this visualization (Supplementary Table 2).

